src/js/actions/BallotActions.js

```diff
@@ -4,7 +4,7 @@ var BallotActions = (function (_Dispatcher) {
   function _BallotActions () { }
 
   _BallotActions.prototype.init = function init () {
-    _Dispatcher.loadEndpoint("voterBallotItemsRetrieve", { use_test_election: true });
```

Comment:
Can these be left in and set to false to be turned off or is the boolean used in another fashion?